TaskList/app/helpers/people_helper.rb

```diff
@@ -0,0 +1,13 @@
+module PeopleHelper
+  def num_incomplete(person_id)
```

Comment:
Love the use of view helper methods! I think some of this code might be better off in the `Person` model class.

Hard to say in this case since there's not a log going on, but I could see breaking this up into a `Person#incomplete_tasks` method and have:
```ruby
def num_incomplete(person_id)
  return Person.find(person_id).incomplete_tasks.length
```